src/rustup-win-installer/msi/Completion.Tests.ps1

```diff
@@ -0,0 +1,36 @@
+if (Get-Command -Name rustup -ErrorAction Stop) {
+	# Enable rustup tab completion
+	rustup completions powershell | Out-String | Invoke-Expression
+}
+
+function Get-Result([string]$Text, [int]$CursorPosition = $Text.Length, [hashtable]$Options) {
+	[System.Management.Automation.CommandCompletion]::CompleteInput($Text, $CursorPosition, $Options).CompletionMatches
+}
+
+Describe 'rustup flags and subcommands' {
+	$result = Get-Result 'rustup '
+	$result.Count | Should Be 23
```

Comment:
I'm worried that doing this exact test might be risky going forward.

Perhaps, if it's possible, something like a "More than 10" type test would be better, and then rather than requiring the specific ordering of content subsequent in this function, that we look for some critical words instead such as install, update, completions, --help, --verbose.  Otherwise I'm really worried that as we adjust the command line interface (which will happen, even if it's just adding stuff) it's going to be hard for any of us who don't know PowerShell to keep track.

Reply:
:+1: probably better to not over-specify.

Reply:
I agree.